static/src/javascripts/projects/common/modules/commercial/ticker.js

```diff
@@ -37,14 +37,16 @@ const increaseCounter = (
     counterElement: HTMLElement
 ) => {
     // Count is local to the parent element
-    count[parentElementSelector] += Math.floor(total / 100);
+    const newCount = count[parentElementSelector] + Math.floor(total / 100);
 
-    counterElement.innerHTML = `${getLocalCurrencySymbol()}${count[
-        parentElementSelector
-    ].toLocaleString()}`;
-    if (count[parentElementSelector] >= total) {
+    if (newCount >= total || newCount <= count[parentElementSelector]) {
```

Comment:
as with the support-frontend PR, I find it hard to tell what this new condition is all about ... can we clarify somehow?